docs/DeveloperGuide.adoc

```diff
@@ -795,6 +1043,104 @@ Private contact detail
 A contact detail that is not meant to be shared with others
 ....
 
+[[add]]
+add
```

Comment:
ideally glossary should be terms used in the address book not easily understood. not sure if command description should go here. you will have the user guide for these